.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -232,6 +232,7 @@ codeclimate
 codeconventions
 codecov
 codehaus
+codepoint
```

Comment:
Can we prevent extending this whitelist by renaming something? `codePoints` already exists in project, so that should be ok. If `CodePointUtil` is causing us to extend whitelist, we can rename it to `CodePointsUtil`.

Reply:
`CodePointUtil` is not a problem
https://github.com/checkstyle/checkstyle/blob/fa18b768c331f5c4d814b2fe250f0b4e0c34bf0f/src/main/java/com/puppycrawl/tools/checkstyle/utils/CodePointUtil.java#L34
I should have made it `code point`

Reply:
Thanks for investigation, please do this.

Reply:
done

Reply:
Resolved.